paperqa/types.py

```diff
@@ -73,7 +77,7 @@ class LLMResult(BaseModel):
 
     id: UUID = Field(default_factory=uuid4)
     session_id: UUID | None = Field(
-        default_factory=cvar_session_id.get,
```

Comment:
I looked into this, it's a `pydantic` issue: https://github.com/pydantic/pydantic/issues/10950

Can you just do a `# type: ignore[arg-type]` here, and link that issue